packages/react-native/project.json

```diff
@@ -27,10 +25,10 @@
           "cp -r packages/react-native/README.md dist/react-native",
           "cp -r LICENSE.package dist/react-native/LICENSE",
           "cp -r packages/react-native/app.plugin.js dist/react-native",
-          "cp packages/react-native/package.dist.json dist/react-native/package.json",
+          "jq '{name, version, description, author, license, homepage, bugs, repository, main: (.main | sub(\"^dist/\"; \"\") + \".js\"), module: (.module | sub(\"^dist/\"; \"\") + \".js\"), \"react-native\": (.main | sub(\"^dist/\"; \"\") + \".js\"), types: (.types | sub(\"^dist/\"; \"\")), codegenConfig, dependencies, peerDependencies}' packages/react-native/package.json > dist/react-native/package.json",
```

Comment:
Considered making this a script, but leaving it inline for now

Reply:
what a monster